src/app/login/Login.types.ts

```diff
@@ -0,0 +1,9 @@
+import { LoginFields } from './loginForm/LoginForm.types';
+
+export type LoginProps = {
+  onLogin: (data: LoginFields) => void;
+  token: string | undefined;
+  isLoading: boolean;
+  isError: boolean;
```

Comment:
Właściwie to ten `isError` możnaby pominąć, prawda? Bo jeśli `error` jest niepusty, to wtedy znaczy że jest error ;)